assets/scss/styles.scss

```diff
@@ -0,0 +1,4 @@
+---
+---
+
+@import "homepage/text-cards"
```

Comment:
should be `@use` cos this is `.scss` no? see [here](https://sass-lang.com/documentation/at-rules/use/)

Reply:
swapped in https://github.com/isomerpages/isomerpages-template/pull/303/commits/fa9db7f89dcd0e3a4d6d00277317e44c285aa042!

Reply:
Lgtm lfg